gtsam/geometry/Similarity3.h

```diff
@@ -180,15 +180,15 @@ class Similarity3: public LieGroup<Similarity3, 7> {
   /// @{
 
   /// Calculate 4*4 matrix group equivalent
-  GTSAM_EXPORT const Matrix4 matrix() const;
+  GTSAM_EXPORT Matrix4 matrix() const;
```

Comment:
GTSAM_EXPORT on class, not methods

Reply:
Done for both Similarity2 and Similarity3